### PR TITLE
update papers

### PR DESCRIPTION
**Pull request by eytan**
## Motivation

(Write your motivation here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/pytorch/botorch/blob/main/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/pytorch/botorch, and link to your PR here.)

### CODE REVIEW

File: docs/papers.md [hunk 1]
```diff
@@ -18,24 +18,52 @@ The main reference for BoTorch is
 
 Here is an incomplete selection of peer-reviewed Bayesian optimization papers that build off of BoTorch:
 
+- [Bayesian Optimization over Discrete and Mixed Spaces via Probabilistic Reparameterization](https://arxiv.org/pdf/2210.10199.pdf). Samuel Daulton, Xingchen Wan, David Eriksson, Maximilian Balandat, Michael A Osborne, Eytan Bakshy. NeurIPS 2022.
+- [Robust Multi-Objective Bayesian Optimization Under Input Noise](https://arxiv.org/pdf/2202.07549.pdf)/ Samuel Daulton, Sait Cakmak, Maximilian Balandat, Michael A Osborne, Enlu Zhou, Eytan Bakshy. ICML 2022.
```
Comment (esantorella) [inline docs/papers.md:22]:
```suggestion
- [Robust Multi-Objective Bayesian Optimization Under Input Noise](https://arxiv.org/pdf/2202.07549.pdf). Samuel Daulton, Sait Cakmak, Maximilian Balandat, Michael A Osborne, Enlu Zhou, Eytan Bakshy. ICML 2022.
```

File: docs/papers.md [hunk 2]
```diff
@@ -18,24 +18,52 @@ The main reference for BoTorch is
...
+
+- [Multi-Objective Bayesian Optimization over High-Dimensional Search Spaces](https://arxiv.org/pdf/2109.10964.pdf)/
```
Comment (esantorella) [inline docs/papers.md:24]:
```suggestion
- [Multi-Objective Bayesian Optimization over High-Dimensional Search Spaces](https://arxiv.org/pdf/2109.10964.pdf).
```

File: docs/papers.md [hunk 3]
```diff
@@ -18,24 +18,52 @@ The main reference for BoTorch is
...
+Samuel Daulton, David Eriksson, Maximilian Balandat, Eytan Bakshy. UAI 2022.
+
+- [Preference Exploration for Efficient Bayesian Optimization with Multiple Outcomes](https://arxiv.org/pdf/2203.11382.pdf).
+Jerry Lin, Raul Astudillo, Peter Frazier, Eytan Bakshy. AISTATS 2022.
+
+- [Look-Ahead Acquisition Functions for Bernoulli Level Set Estimation](https://arxiv.org/pdf/2203.09751.pdf).
+Benjamin Letham, Eytan Bakshy, Michael Shvartsman. AISTATS 2022.
+
+- [GIBBON: General-purpose Information-Based Bayesian OptimisatioN](https://jmlr.org/papers/volume22/21-0120/21-0120.pdf). Henry B. Moss, David S. Leslie, Javier Gonzalez, Paul Rayson. JMLR 2021.
```
Comment (esantorella) [inline docs/papers.md:33]:
```suggestion
- [GIBBON: General-purpose Information-Based Bayesian Optimisation](https://jmlr.org/papers/volume22/21-0120/21-0120.pdf). Henry B. Moss, David S. Leslie, Javier Gonzalez, Paul Rayson. JMLR 2021.
```

File: docs/papers.md [hunk 4]
```diff
@@ -18,24 +18,52 @@ The main reference for BoTorch is
...
+
+- [Bayesian Optimization with High-Dimensional Outputs](https://arxiv.org/pdf/2106.12997.pdf).
+Wesley Maddox, Max Balandat, Andrew Gordon Wilson, Eytan Bakshy. NeurIPS 2021.
+
+- [Conditioning Sparse Variational Gaussian Processes
+for Online Decision-making](https://proceedings.neurips.cc/paper/2021/file/325eaeac5bef34937cfdc1bd73034d17-Paper.pdf). Maddox, Wesley J., Samuel Stanton, and Andrew G. Wilson. NeurIPS 2021
```
Comment (esantorella) [inline docs/papers.md:39]:
```suggestion
for Online Decision-making](https://proceedings.neurips.cc/paper/2021/file/325eaeac5bef34937cfdc1bd73034d17-Paper.pdf). Wesley J. Maddox, Samuel Stanton, and Andrew G. Wilson. NeurIPS 2021
```

File: docs/papers.md [hunk 5]
```diff
@@ -18,24 +18,52 @@ The main reference for BoTorch is
...
-- [High-Dimensional Bayesian Optimization via Nested Riemannian Manifolds](https://proceedings.neurips.cc/paper/2020/hash/f05da679342107f92111ad9d65959cd3-Abstract.html). Noémie Jaquier, Leonel Rozo. NeurIPS 2020.
-- [Efficient Nonmyopic Bayesian Optimization via One-Shot Multi-Step Trees](https://proceedings.neurips.cc/paper/2020/hash/d1d5923fc822531bbfd9d87d4760914b-Abstract.html). Shali Jiang, Daniel Jiang, Maximilian Balandat, Brian Karrer, Jacob Gardner, Roman Garnett. NeurIPS 2020.
 - [Re-Examining Linear Embeddings for High-Dimensional Bayesian Optimization](https://proceedings.neurips.cc/paper/2020/hash/10fb6cfa4c990d2bad5ddef4f70e8ba2-Abstract.html). Ben Letham, Roberto Calandra, Akshara Rai, Eytan Bakshy. NeurIPS 2020.
 - [PareCO: Pareto-aware Channel Optimization for Slimmable Neural Networks
 ](https://arxiv.org/abs/2007.11752). Ting-Wu Chin, Ari S. Morcos, Diana Marculescu. ICML 2020 Workshop on Real World Experiment Design and Active Learning.
 - [High-Dimensional Bayesian Optimization with Sparse Axis-Aligned Subspaces
 ](https://proceedings.mlr.press/v161/eriksson21a.html). David Eriksson, Martin Jankowiak. UAI 2021.
```
Comment (esantorella) [inline docs/papers.md:58]:
```suggestion
- [High-Dimensional Bayesian Optimization with Sparse Axis-Aligned Subspaces](https://proceedings.mlr.press/v161/eriksson21a.html). David Eriksson, Martin Jankowiak. UAI 2021.
```

File: docs/papers.md [hunk 6]
```diff
@@ -18,24 +18,52 @@ The main reference for BoTorch is
...
 - [Bayesian Optimization over Permutation Spaces](https://arxiv.org/abs/2112.01049). Aryan Deshwal, Syrine Belakaria, Janardhan Rao Doppa, Dae Hyun Kim. AAAI 2021.
 - [GIBBON: General-purpose Information-Based Bayesian Optimisation](https://jmlr.org/papers/v22/21-0120.html). Henry B. Moss, David S. Leslie, Javier Gonzalez, Paul Rayson. JMLR, 2021.
```
Comment (esantorella) [inline docs/papers.md:60]:
```suggestion
```
duplicated

File: docs/papers.md [hunk 7]
```diff
@@ -18,24 +18,52 @@ The main reference for BoTorch is
...
+Jerry Lin, Raul Astudillo, Peter Frazier, Eytan Bakshy. AISTATS 2022.
+
+- [Look-Ahead Acquisition Functions for Bernoulli Level Set Estimation](https://arxiv.org/pdf/2203.09751.pdf).
+Benjamin Letham, Eytan Bakshy, Michael Shvartsman. AISTATS 2022.
+
+- [GIBBON: General-purpose Information-Based Bayesian OptimisatioN](https://jmlr.org/papers/volume22/21-0120/21-0120.pdf). Henry B. Moss, David S. Leslie, Javier Gonzalez, Paul Rayson. JMLR 2021.
+
+- [Bayesian Optimization with High-Dimensional Outputs](https://arxiv.org/pdf/2106.12997.pdf).
+Wesley Maddox, Max Balandat, Andrew Gordon Wilson, Eytan Bakshy. NeurIPS 2021.
+
```
Comment (esantorella) [inline docs/papers.md:37]:
```suggestion
```
duplicated

**Review by esantorella — APPROVED** [review]
Approving to unblock, but I suggested a few changes -- removing two duplicated papers and some minor whitespace and punctuation edits

**Comment by eytan** [comment]
OK, lgtm.  The actual paper title for gibbon does end with a capital N, but since readers might think its a typo, we might as well not capitalize it.   Can you help with merging it in?

**Pull request closed without merging**